udata/frontend/markdown.py

```diff
@@ -40,7 +40,7 @@ def __call__(self, stream):
             styles=current_app.config['MD_ALLOWED_STYLES'],
             strip_comments=False)
         # Turn markdown to HTML.
-        ast = self.parser.parse(stream)
+        ast = self.parser().parse(stream)
```

Comment:
It looks like we can keep a single instance of the `renderer` so I would keep it for performance matters.
The new instance of the parser is required in order to deal with thread-safe issues we have when multiple commonmark conversions are issued at the same time. Maybe we should declare the bug upstream but it's hard to reproduce/provide a test for this.